docs/source/llms/langchain/index.rst

```diff
@@ -550,3 +550,44 @@ string response, which can be compared to the `"ground_truth"` column.
 
 For a complete example of a LangGraph model that works with this evaluation example, see the 
 `MLflow LangGraph blog <https://mlflow.org/blog/langgraph-model-from-code>`_.
+
+How to control whether my input is converted to List[langchain.schema.BaseMessage] in PyFunc predict?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If user provides inputs like ``{"messages": [{"role": "user", "content": "some_question"}]}``, MLflow internally tries to
+convert the input to List[langchain.schema.BaseMessage] like ``[HumanMessage(content="some_question")]``. If the model is
+AgentExecutor or the model's input_schema contains "messages" field, then MLflow doesn't convert the inputs.
+There're some edge cases where user wants to provide the input as is, in such cases user can set the environment variable
```

Comment:
Can we simply this? For example:

By default, MLflow converts the input in the chat format (<example>) to a ``List[langchain.schema.BaseMessage]`` object for certain model types. To disable this behavior, set <env var> to ``False`` as demonstrated below.